[Bug] r.in.pdal: compilation broken by PDAL 2.4.0

**Describe the bug**
Compilation with PDAL 2.4.0 fails.

Error log:

```
/usr/bin/clang++  -g -pipe -stdlib=libc++ -arch arm64 -std=c++11   -I/Volume/dev/grass/dist.aarch64-apple-darwin21.4.0/include -I/Volume/dev/grass/dist.aarch64-apple-darwin21.4.0/include   -I/opt/local/include -I/opt/local/include -pipe -I/opt/local/libexec/openssl3/include -Os -DNDEBUG -I/opt/local/libexec/openssl3/include -isystem/opt/local/include -stdlib=libc++ -std=c++11 -DPACKAGE=\""grassmods"\"  -I/opt/local/include/postgresql13 -I/opt/local/lib/proj8/include -I/opt/local/include -I/opt/local/include -I/opt/local/include/libxml2 -I/Volume/dev/grass/dist.aarch64-apple-darwin21.4.0/include -I/Volume/dev/grass/dist.aarch64-apple-darwin21.4.0/include -DRELDIR=\"raster/r.in.pdal\" -o OBJ.aarch64-apple-darwin21.4.0/info.o -c info.cpp
info.cpp:37:25: error: call to implicitly-deleted copy constructor of 'pdal::LasHeader'
        pdal::LasHeader las_header = las_reader.header();
                        ^            ~~~~~~~~~~~~~~~~~~~
/opt/local/include/pdal/io/LasHeader.hpp:295:30: note: copy constructor of 'LasHeader' is implicitly deleted because field 'd' has a deleted copy constructor
    std::unique_ptr<Private> d;
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/__memory/unique_ptr.h:213:3: note: copy constructor is implicitly deleted because 'unique_ptr<pdal::LasHeader::Private>' has a user-declared move constructor
  unique_ptr(unique_ptr&& __u) _NOEXCEPT
  ^
info.cpp:98:25: error: call to implicitly-deleted copy constructor of 'pdal::LasHeader'
        pdal::LasHeader las_header = las_reader.header();
                        ^            ~~~~~~~~~~~~~~~~~~~
/opt/local/include/pdal/io/LasHeader.hpp:295:30: note: copy constructor of 'LasHeader' is implicitly deleted because field 'd' has a deleted copy constructor
    std::unique_ptr<Private> d;
                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/__memory/unique_ptr.h:213:3: note: copy constructor is implicitly deleted because 'unique_ptr<pdal::LasHeader::Private>' has a user-declared move constructor
  unique_ptr(unique_ptr&& __u) _NOEXCEPT
  ^
info.cpp:103:19: warning: 'operator<<' is deprecated [-Wdeprecated-declarations]
        std::cout << las_header;
                  ^
/opt/local/include/pdal/io/LasHeader.hpp:68:3: note: 'operator<<' has been explicitly marked deprecated here
[[deprecated]] std::ostream& operator<<(std::ostream& ostr, const LasHeader& h);
  ^
1 warning and 2 errors generated.
make: *** [OBJ.aarch64-apple-darwin21.4.0/info.o] Error 1

```

**System description (please complete the following information):**

- Operating System: All
- GRASS GIS version:  All (but tested on main)

<!---

- details about further software components
    - run `g.version -rge` in a GRASS GIS terminal session or check in
      the GUI menu "Help > About"
    - run `python3 -c "import sys, wx; print(sys.version); print(wx.version())"`
      to print the Python and wxPython version numbers
--->

--- a/raster/r.in.pdal/info.cpp
+++ b/raster/r.in.pdal/info.cpp
@@ -34,7 +34,7 @@ void get_extent(struct StringList *infiles, double *min_x, double *max_x,
         pdal::LasReader las_reader;
         las_reader.setOptions(las_opts);
         las_reader.prepare(table);
-        pdal::LasHeader las_header = las_reader.header();
+        const pdal::LasHeader& las_header = las_reader.header();
         if (first) {
             *min_x = las_header.minX();
             *min_y = las_header.minY();
@@ -95,7 +95,7 @@ void print_lasinfo(struct StringList *infiles)
         pdal::LasReader las_reader;
         las_reader.setOptions(las_opts);
         las_reader.prepare(table);
-        pdal::LasHeader las_header = las_reader.header();
+        const pdal::LasHeader& las_header = las_reader.header();
         pdal::PointLayoutPtr point_layout = table.layout();
         const pdal::Dimension::IdList & dims = point_layout->dims();
 

